<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="add_collection_button">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="list_collections">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Все коллекции</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
